addons/beehave/debug/debugger.gd

```diff
@@ -2,6 +2,8 @@
 extends EditorDebuggerPlugin
 
 const DebuggerTab := preload("debugger_tab.gd")
+const BeehaveUtils := preload("res://addons/beehave/utils/utils.gd")
```

Comment:
This avoids issues at initial load in case the global class cache might not have initialized `BeehaveUtils` yet.